src/net_processing.cpp

```diff
@@ -2924,6 +2924,14 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
             return;
         }
 
+        // If we are in IBD, exit from unrequested transaction processing early to save bandwidth
+        // and mitigate potential DoS risks.
+        if (!pfrom.HasPermission(PF_RELAY) && ::ChainstateActive().IsInitialBlockDownload())
```

Comment:
```suggestion
        if (!pfrom.HasPermission(PF_RELAY) && m_chainman.ActiveChainstate().IsInitialBlockDownload()) {
```

style-nit: Please don't use the deprecated wrappers. This is going to silently conflict with ongoing work to remove them

Also clang-format for new code